### DISCUSSION TITLE
Integrar ferramentas de testes automatizados para acessibilidade

### DISCUSSION THREAD

**Discussion by andrewarosario** [techexpertspro/portal-ascepa#14] [Ideas]
Visto que o projeto tem como prioridade a acessibilidade, esta discussão é para propormos ideias para aprimorar essa questão dentro dos testes automatizados.
O foco não seria somente criar testes para garantir as funcionalidades, mas também para garantir que os componentes do projeto estejam acessíveis.

### Ferramentas para análise
- [axe](https://www.npmjs.com/package/jest-axe): Uma ferramenta abrangente que se integra a várias estruturas de teste. Possui lib para o jest.
- [Lighthouse](https://chromewebstore.google.com/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk?hl=en): Uma ferramenta do Google que fornece uma ampla gama de auditorias, incluindo acessibilidade.
- [Wave API](https://wave.webaim.org/api/): Fornece relatórios detalhados de acessibilidade e pode ser usada programaticamente.
Tenon.io: Oferece uma API para testes de acessibilidade.

**Comment by brunoredes** [comment]
Podemos também cogitar o uso do axe-devtools [Extensão chromium based](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?pli=1) | [Site axe devtools](https://www.deque.com/axe/devtools/)

E também fazer as navegações dentro do sistema com algum leitor de tela ligado, como os:
[NVDA](https://www.nvaccess.org/download/)
[Orca - Gnome](https://help.gnome.org/users/orca/stable/)
[JAWS](https://www.tecassistiva.com.br/catalogo/jaws/)

**Comment by Flamilani** [comment]
tem outras ferramentas e extensões para teste de acessibilidade:
[Axe for Designers](https://www.deque.com/axe/design-beta/), 
[Access Monitor](https://accessmonitor.acessibilidade.gov.pt/), 
[Screen Reader](https://chromewebstore.google.com/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=pt-BR)

Guia de criação de aplicativos do Angular acessíveis:
https://codelabs.developers.google.com/angular-a11y?hl=pt-br#0

Checklist de Acessibilidade Web:
https://dequeuniversity.com/checklists/web/

**Comment by andrewarosario** [comment]
Criado PR #80 para integrar o jest-axe no projeto.
